DockerOrchestration/ABCIApp/Dockerfile.Dummy

```diff
@@ -0,0 +1,15 @@
+FROM golang:1.8
```

Comment:
@ebuchman  Those dockerfiles will be cleaned up, I just wanted an easy way to test multiple ABCIs in this setup.

@melekes The reason why I have 2 dockerfiles is that I generally think that it is a better idea to run the core and abci nodes separately. 
It allows us to more easily scale the ABCI parts, since we can orchestrate multiple ABCI apps in front of a single core node. 
Furthermore, it makes the separation of interfaces easier. 
Lastly, I think most people will run the ABCI apps and core nodes as separate processes since it seems easier than running them in-process and hence my idea was to emulate it with the dummy apps. :-)